staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -260,9 +263,15 @@ func (c *Config) ApplySecureServingOptions(secureServing *options.SecureServingO
 		return c, nil
 	}
 
+	ipMax := c.MaxRequestsInFlight / 2
+	if ipMax > c.MaxMutatingRequestsInFlight/2 {
+		ipMax = c.MaxMutatingRequestsInFlight / 2
+	}
+	max := c.MaxRequestsInFlight + c.MaxMutatingRequestsInFlight
```

Comment:
Yes, they consume some of the maximum connection count. However they still do not count against the maximum number of in-flight requests. So the documentation is still technically correct. Do you feel like we should not count requests matching the regex from counting against the connection limit? This is actually fairly hairy to handle. Not only would we need to worry about decrementing the limit when we got a match, (make sure we do not over-decrement if we allow multiple requests per connection) but we would also then need to worry about re-incrementing the count once that request was finished.